rundeckapp/grails-app/views/menu/_jobPickerModalPagination.gsp

```diff
@@ -1,21 +1,31 @@
 <g:if test="${max && max < total && jobsListSize < total}">
-    <div id="modal-pagination" style="display: none">
-        <span>
```

Comment:
This can be removed. In my PR I had included the offset, total and max attributes but that shouldn't be here.

Reply:
Offset values span has  been removed